plugins/Hosting/js/sagas/saga.js

```diff
@@ -52,14 +52,25 @@ function *announceHost(action) {
 	}
 }
 
+// bytesToStorage converts a BigNumber of GB to a valid size of storage, in
+// bytes, rounded to the nearest 256MiB (64 * SectorSize).
+const bytesToStorageBytes = (bytes) => {
+	const roundedBytes = bytes.minus(bytes.modulo(2.6843546e8))
```

Comment:
kind of a weird number. Might as well just write out `268435456` or `64 * 4194304`